src/EventListener/InitializeSystemListener.php

```diff
@@ -138,10 +138,12 @@ public function onConsoleCommand()
     protected function setConstants($mode, $route)
     {
         // The constants are deprecated and will be removed in version 5.0.
+        $time = microtime(true);
```

Comment:
Of course it should be but we cannot deprecate anything as long as there's no alternative to be used :)